build.gradle

```diff
@@ -231,6 +231,7 @@ project(':iceberg-core') {
       exclude group: 'org.slf4j', module: 'slf4j-log4j12'
     }
 
+    testRuntime "com.h2database:h2"
```

Comment:
I think `testCompile` is good enough, if we want to upgrade gradle we should upgrade all syntax at the same time.

Reply:
updated